esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -55,17 +53,19 @@ def _cube_to_GridInfo(cube):
     )
 
 
-# def _regrid_along_dims(regridder, data, src_dim, mdtol):
-#     # Before regridding, data is transposed to a standard form.
-#     # This will be done either with something like the following code
-#     # or else done within the regridder by specifying args.
-#     # new_axes = list(range(len(data.shape)))
-#     # new_axes.pop(src_dim)
-#     # new_axes.append(src_dim)
-#     # data = ma.transpose(data, axes=new_axes)
-#
-#     result = regridder.regrid(data, mdtol=mdtol)
-#     return result
+def _regrid_along_dims(regridder, data, src_dim, mdtol):
```

Comment:
Is the name of this function as clear as it could be?  what do we mean by "along dims"?  is `src_dim` the same as `mesh_dim` name used elsewhere?